user_templates/mcu_defaults/mmu/BTT_MMB_CAN_v1.0.cfg

```diff
@@ -0,0 +1,52 @@
+
+#-----------------------------------------------#
+#### BTT MMB CAN board MCU definition ###########
+#-----------------------------------------------#
+
+[mcu mmu]
+##--------------------------------------------------------------------
+serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+# canbus_uuid: change-me-to-the-correct-canbus-id
+##--------------------------------------------------------------------
+
+# If you want to override the wiring of the BTT MMB CAN board, keep in mind that this
+# board is defined using the "mmu" name. So you should use "pin: mmu:PIN_NAME"
+# in your own overrides.cfg files.
+
+[include config/mcu_definitions/mmu/BTT_MMB_CAN_v1.0.cfg] # Do not remove this line
+[board_pins mmu_mcu]
+mcu: mmu
+aliases:
+    MMU_GEAR_STEP=MCU_M1_STEP  , MMU_GEAR_DIR=MCU_M1_DIR  , MMU_GEAR_ENABLE=MCU_M1_EN  , MMU_GEAR_UART=MCU_M1_UART  ,
+    MMU_GEAR_DIAG=MCU_STP1     , # Shared with MCU_STP1 !!!
+    MMU_SEL_STEP=MCU_M2_STEP   , MMU_SEL_DIR=MCU_M2_DIR   , MMU_SEL_ENABLE=MCU_M2_EN   , MMU_SEL_UART=MCU_M2_UART   ,
+    MMU_SEL_DIAG=MCU_STP2      , # Shared with MCU_STP2 !!!
+
+    MMU_SEL_ENDSTOP=MCU_STP11  ,
+
+    MMU_SERVO=MCU_MOT          ,
+    MMU_ENCODER=MCU_SENSOR     ,
+    MMU_NEOPIXEL=MCU_RGB       ,
+#    MMU_GATE_SENSOR=MCU_STP1  , # (if not GEAR DIAG!)
+
+    MMU_PRE_GATE_0=MCU_STP3    ,
+    MMU_PRE_GATE_1=MCU_STP4    ,
+    MMU_PRE_GATE_2=MCU_STP5    ,
+    MMU_PRE_GATE_3=MCU_STP6    ,
+    MMU_PRE_GATE_4=MCU_STP7    ,
+    MMU_PRE_GATE_5=MCU_STP8    ,
+    MMU_PRE_GATE_6=MCU_STP9    ,
+    MMU_PRE_GATE_7=MCU_STP10   ,
+    MMU_PRE_GATE_8=""          ,
+    MMU_PRE_GATE_9=""          ,
+    MMU_PRE_GATE_10=""         ,
+    MMU_PRE_GATE_11=""         ,
```

Comment:
do we need these when there is no more "hardware" connections available on the MMB?

Reply:
I've already commented them in my current local working directory. I'll push it in a couple of minutes (when I finish the documentation). I want to keep them here commented in case the user would like to remap: it will be easier to know that they exists. But we can fuly delete them if you prefer.